README.md

```diff
@@ -22,7 +22,13 @@ npm run type-check
 ## Install your dependencies
 
 ```sh
-npm install --save-dev babel-cli@7.0.0-beta.0 babel-plugin-transform-class-properties@7.0.0-beta.0 babel-plugin-transform-object-rest-spread@7.0.0-beta.0 babel-preset-env@2.0.0-beta.0 babel-preset-typescript@7.0.0-beta.0 typescript@2.5.2
+npm install --save-dev typescript@2.6.2
+npm install --save-dev @babel/core@7.0.0-beta.32
+npm install --save-dev @babel/cli@7.0.0-beta.32
+npm install --save-dev @babel/plugin-proposal-class-properties@7.0.0-beta.32
```

Comment:
should we make a separate preset (or something of the sort, maybe even an option) that includes the proposals that typescript also supports?